cosi/protocol/gen_tree.go

```diff
@@ -0,0 +1,114 @@
+package protocol
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/dedis/onet"
+	"github.com/dedis/onet/network"
+)
+
+// GenTrees will create a given number of subtrees of the same number of nodes.
+// Each generated subtree will have the same root.
+// Each generated tree have a root with one child (the subleader)
+// and all other nodes in the tree will be the subleader children.
+// NOTE: register being not implementable with the current API could hurt the scalability tests
+func GenTrees(roster *onet.Roster, nNodes, nSubtrees int) ([]*onet.Tree, error) {
```

Comment:
It seems like this code could be in onet/treenode.go. Not saying it should be, just checking your opinion... is it general purpose, or cosi-only?

Reply:
This is somewhat cosi-only, as it only generates 3-level trees. We may, however, simplify the code by making use of existing onet tree generation functions. I will add a comment.